ouroboros-network/test/Test/ChainGenerators.hs

```diff
@@ -94,14 +94,35 @@ tests = testGroup "Chain"
   ]
 
 
+instance Arbitrary BlockNo where
```

Comment:
We have `Arbitrary` instances for `BlockNo` and `SlotNo` in `Test.Util.Orphans.Arbitrary` already. Would it make sense to unify these?

Reply:
It would yes. We should move more of our testing stuff into our testing library, so it can be shared between the network testsuite and reused in the consensus testsuite.